cannot convert nil to type _Ctype_jobject

When I'm trying to build for android I'm getting the following errors since updating to go 1.11:

```
# github.com/hajimehoshi/oto
../../../hajimehoshi/oto/player_android.go:205:31: cannot convert nil to type _Ctype_jobject
../../../hajimehoshi/oto/player_android.go:280:18: cannot convert nil to type _Ctype_jobject
../../../hajimehoshi/oto/player_android.go:293:15: cannot use nil as type _Ctype_jobject in assignment
```

Could be due to the changes to the changes to the way cog interacts with the jobject hierarchy in Java's JNI interface.

From the [release notes](https://golang.org/doc/go1.10#cgo) (this is for 1.10 but then they were warnings now they're errors): 

"These types must be uintptr on the Go side because they would otherwise confuse the Go garbage collector; they are sometimes not really pointers but data structures encoded in a pointer-sized integer. Pointers to Go memory must not be stored in these uintptr values.

Because of this change, values of the affected types need to be zero-initialized with the constant 0 instead of the constant nil. Go 1.10 provides gofix modules to help with that rewrite:

go tool fix -r cftype <pkg>
go tool fix -r jni <pkg>

--- a/player_android.go
+++ b/player_android.go
@@ -202,7 +202,7 @@ func newPlayer(sampleRate, channelNum, bytesPerSample, bufferSizeInBytes int) (*
 	runtime.SetFinalizer(p, (*player).Close)
 
 	if err := app.RunOnJVM(func(vm, env, ctx uintptr) error {
-		audioTrack := C.jobject(nil)
+		audioTrack := C.jobject(0)
 		bufferSize := C.int(bufferSizeInBytes)
 		if msg := C.initAudioTrack(C.uintptr_t(vm), C.uintptr_t(env),
 			C.int(sampleRate), C.int(channelNum), C.int(bytesPerSample),

--- a/player_android.go
+++ b/player_android.go
@@ -277,7 +277,7 @@ func (p *player) TryWrite(data []byte) (int, error) {
 }
 
 func (p *player) Close() error {
-	if p.audioTrack == nil {
+	if p.audioTrack == 0 {
 		return nil
 	}
 
@@ -290,6 +290,6 @@ func (p *player) Close() error {
 		return nil
 	})
 
-	p.audioTrack = nil
+	p.audioTrack = 0
 	return err
 }

